src/NugetReferenceResolver/RuntimeGraph.cs

```diff
@@ -0,0 +1,926 @@
+﻿using Microsoft.DotNet.PlatformAbstractions;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NugetReferenceResolver
+{
+    public class RuntimeGraph
+    {
+        private const string RuntimeCompatibility = @"
```

Comment:
This whole graph is already available on `Microsoft.Extensions.DependencyModel.DependencyContext.RuntimeGraph.`